public/controllers/management/components/upload-files.js

```diff
@@ -284,6 +303,13 @@ export class UploadFiles extends Component {
           <EuiTitle size="m">
             <h1>{`Upload ${this.props.resource}`}</h1>
           </EuiTitle>
+          <EuiCheckbox
+            className='wz-margin-top-10 wz-margin-bottom-10'
+            id="overwrite"
+            label="Activate overwriting"
+            checked={this.state.overwrite}
+            onChange={(e) => this.activeOverwrite(e)}
```

Comment:
Would `onChange={this.activeOverwrite}` work?

Reply:
If the logic of that method, uses the `this` keyword, we could put:

```
onChange={this.activeOverwrite}
```

but it is required to bind the `this` context to the method. In the class constructor, we should put:
```
this.activeOverwrite = this.activeOverwrite.bind(this);
```

Reply:
Solved